lib/libtlspool.c

```diff
@@ -39,7 +39,7 @@
 #define BUFSIZE 4096
 #define random rand
 #define srandom srand
-
+#define IPPROTO_SCTP 132
```

Comment:
Please don't overrule known settings, just compensate when something is missing:

```
/* Windows supports SCTP but fails to define this IANA-standardised symbol: */
#ifndef IPPROTO_SCTP
#define IPPROTO_SCTP 132
#endif
```

Reply:
aangepast